src/pages/home/Resources.tsx

```diff
@@ -7,46 +7,90 @@ import TextGroup from "../../components/TextGroup";
 export default () => {
   return (
     <Section>
-      <SideBySide
-        left={() => (
-          <TextGroup>
-            <h2>Resources</h2>
-            <h3>CONTACT ORGANIZERS</h3>
-            <p>
-              Email :
-              <a href="mailto:organizers@usereact.nyc">
-                organizers@usereact.nyc
-              </a>
-            </p>
-            <p>
-              Phone : <span className="span-text-link">914-297-8806</span>
-            </p>
-            <h3>LOCAL RESOURCES</h3>
-            <p>
-              Local Law Enforcement:{" "}
-              <span className="span-text-link">emergency 911</span>
-            </p>
-            <p>
-              Local Law Enforcement:{" "}
-              <span className="span-text-link">non-emergency 311</span>
-            </p>
-            <p>
-              Local sexual assault hotline:{" "}
-              <span className="span-text-link">800-621-HOPE(4673)</span> or dial{" "}
-              <span className="span-text-link">311</span>
-            </p>
-            <p>
-              Local medical:{" "}
-              <span className="span-text-link">emergency 911</span>
-            </p>
-            <p>
-              Local medical:{" "}
-              <span className="span-text-link">non-emergency 311</span>
-            </p>
-          </TextGroup>
-        )}
-        right={() => <img src={nycTaxiStreet} />}
-      />
+      <div className="resources-container-desktop">
```

Comment:
I left a TODO in css for this. I don't love this solution of show one for mobile then hide it and show the other for desktop, but i didn't want to wreck the side by side component.  And my attempts at modifying it were getting long winded.